static/js/apps/biomed_nl/app.tsx

```diff
@@ -86,23 +102,125 @@ export function App(): ReactElement {
   const [answer, setAnswer] = useState<DisplayedAnswer>(null);
   const [showLoading, setShowLoading] = useState<boolean>(false);
 
+  // A loading spinner that shows new biomed pun every minute
+  function SpinnerWithText(): ReactElement {
+    const [showMessage, setShowMessage] = useState(false);
+    const [message, setMessage] = useState("");
+    const [messageIndex, setMessageIndex] = useState(0);
+
+    const customMessages = useMemo(
```

Comment:
(prem requested biomed puns to be shown on loading)

Reply:
fun! but this can just be a global constant? doesn't need to be a memo?

Reply:
done